application/server/server_smoke_test.go

```diff
@@ -136,11 +144,12 @@ func Test_SmokeWorkspaceScan(t *testing.T) {
 			file2:                "maven-compat/src/test/java/org/apache/maven/repository/legacy/LegacyRepositorySystemTest.java",
 			useConsistentIgnores: true,
 			hasVulns:             true,
+			endpoint:             "https://api.dev.snyk.io",
 		},
 	}
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			runSmokeTest(t, tc.repo, tc.commit, tc.file1, tc.file2, tc.useConsistentIgnores, tc.hasVulns, "")
+			runSmokeTest(t, tc.repo, tc.commit, tc.file1, tc.file2, tc.useConsistentIgnores, tc.hasVulns, tc.endpoint)
```

Comment:
Although we were configuring the endpoint for the `OSS and Code with V1 endpoint` test we weren't passing it down to the function and so always using the non-v1 endpoint.